### ISSUE TITLE
Issue: Docker Commands Compatibility Issue in Windows

### ISSUE THREAD

**Issue by Cauatn** [briefercloud/briefer#7]
I encountered some issues while trying to execute the Docker commands to run the project. It's easily fixable because it's just a matter of replacing the Unix system "\\" with the PowerShell equivalent "`". It would be helpful to mention this nuance somewhere in the documentation, like in the README file.

**Example:**

- **Unix System:**

  ```sh
  docker run -d \
  -p 3000:3000 \
  -p 8080:8080 \
  -v briefer_psql_data:/var/lib/postgresql/data \
  -v briefer_jupyter_data:/home/jupyteruser \
  -v briefer_briefer_data:/home/briefer \
  briefercloud/briefer
  ```

- **PowerShell:**

  ```bash
  docker run -d `
  -p 3000:3000 `
  -p 8080:8080 `
  -v briefer_psql_data:/var/lib/postgresql/data `
  -v briefer_jupyter_data:/home/jupyteruser `
  -v briefer_briefer_data:/home/briefer `
  briefercloud/briefer
  ```
or something like that to avoid some common issues in this repo

**Comment by lucasfcosta** [comment]
Hey @Cauatn! Thank you for the issue!

We'd definitely like to fix this, would you like to submit a pull-request improving our docs? We'd love to have you in the contributors list!

All that we'd need is that you add an extra block to the README like this:

---

When running on Windows' PowerShell, it might be necessary to add an extra ` to the end of each line instead of the `\`, like this:

```bash
docker run -d `
-p 3000:3000 `
-p 8080:8080 `
-v briefer_psql_data:/var/lib/postgresql/data `
-v briefer_jupyter_data:/home/jupyteruser `
-v briefer_briefer_data:/home/briefer `
briefercloud/briefer
```

**Comment by fsntico** [comment]
how to update or install new pip packages?

**Comment by vieiralucas** [comment]
Hi @fsntico you can simply run `!pip install my_package --upgrade` inside a Python block, then restart the environment (there's a button at the bottom right).

By the way this is kinda of an unrelated question with this issue, you got lucky I found this, next time I recommend you to ask at the #support channel of our slack channel or open a new issue.

Thank you and let me know if you still need help installing/updating packages.

**Comment by vieiralucas** [comment]
Closed by #8
